src/components/features/payments/hooks/useCardPasswordValidation.ts

```
@@ -0,0 +1,56 @@
+  const updateCardError = ({
+    errorType,
+    isError,
+  }: {
+    errorType: ErrorType;
+    isError: boolean;
+  }) => {
+    if (isError) {
+      setErrorType((prev) => [...prev, errorType]);
+    } else {
+      setErrorType((prev) =>
+        prev.filter((errorType) => errorType !== errorType)
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)